lib/fastlane/plugin/revenuecat_internal/actions/update_hybrids_versions_file_action.rb

```diff
@@ -19,9 +19,25 @@ def self.run(params)
         ios_version = Helper::UpdateHybridsVersionsFileHelper.get_ios_version_for_hybrid_common_version(hybrid_common_version)
         UI.message("Obtained ios version #{ios_version} for PHC version #{hybrid_common_version}")
 
+        billing_client_version = Helper::UpdateHybridsVersionsFileHelper.get_android_billing_client_version(android_version)
+        UI.message("Obtained android billing client version #{billing_client_version} for PHC version #{hybrid_common_version}")
+
         File.open(versions_file_path, 'r+') do |file|
           lines = file.each_line.to_a
-          lines.insert(2, "| #{new_sdk_version} | #{ios_version} | #{android_version} | #{hybrid_common_version} |\n")
+          if lines[0].split('|').length <= 6
+            lines[0] = "#{lines[0].strip} Android Billing Client version |\n"
+            lines[1] = "#{lines[1].strip}--------------------------------|\n"
```

Comment:
Does it make sense to add the StoreKit version(s) as well? (I suppose a single SDK version can support one or multiple StoreKit versions, so that might make things more complicated.)

Reply:
Yeah... with version 5.0 of the iOS sdk we switched the default to StoreKit 2, but both should still be supported, so I don't think it provides as much value IMO. But we can add it later if needed!

Reply:
Yup, sounds reasonable!